### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by eamador1**
Since the last code review, the following was done:
___

- Remove all hardcoded items from the tasks array ✅
- Create a new JavaScript file for the new functionality ✅
- Implement a function for adding a new task (add a new element to the array) ✅
- Implement a function for deleting a task (remove an element from the array) ✅
- Implement a function for editing task descriptions ✅
- By default new tasks should have the property completed set to false and the property index set to the value of the new array length ✅
- Deleting a task should update all remaining items' indexes ✅
- All changes to the To Do List should be saved in local storage ✅

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    description: 'Take a shower',
+    completed: 'false',
+    index: '1',
+  },
+  {
+    description: 'Breakfast',
+    completed: 'false',
+    index: '2',
+  },
+];
```
Comment (Strangeal) [inline src/index.js]: - [ ] Great job so far on implementing all the necessary functionalities mentioned in the project requirements. However, I noticed that a few parts of the projects haven't been fulfilled yet. To completely fulfill all the requirements for this milestone kindly consider the below suggestions:
  - [ ] Kindly adhere to the project requirements that state that all `hardcoded` tasks should be removed from your codebase. 
  - [ ] Consider adhering to the requirements that state a new file should be created to implement all the functionalities of this milestone. See below for more details on the project requirements: 
  
Screenshot 2023-08-18 at 20 25 09


Great job once again. Happy coding!😊🚀🍀

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @eamador1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Pixel Perfect Design ✅ 
- Professional `README` template ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eamador1** [comment]
Since the last code review, the following changes were made:
___

- Remove all hardcoded items from the task array ✅
- Create a new JavaScript file for new functionalities ✅

**Review by Efiamotu-1 — APPROVED** [review]
Hi @eamador1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

image

## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**